gravity.sh

```diff
@@ -535,11 +535,10 @@ gravity_ParseUserDomains() {
   if [[ ! -f "${blacklistFile}" ]]; then
     return 0
   fi
-
-  mv "${blacklistFile}" "${blackList}.tmp"
+  
   # Copy the file over as /etc/pihole/black.list so dnsmasq can use it
-  mv "${blackList}.tmp" "${blackList}" 2> /dev/null || \
-    echo -e "\\n  ${CROSS} Unable to move ${blackList##*/}.tmp to ${piholeDir}"
+  cp "${blacklistFile}" "${blackList}" 2> /dev/null || \
```

Comment:
@DL6ER this should fix it. I can't set you to review though!

Reply:
I tested this. It works. Feel free to approve (it is my PR so I cannot) and merge. However, note that the FTLDNS PR needs to get merged first such that it can read the new format.